### DISCUSSION TITLE
Adblock isn't working

### DISCUSSION THREAD

**Discussion by matheusbc22** [pear-devs/pear-desktop#87] [Q&A] [answered]
Today the ad block stopped working, idk how, just stopped, i get ads every first song after open the app.

**Comment by th-ch** [comment, marked as answer]
Hey @matheusbc22, thanks for the feedback! It might be related to https://github.com/th-ch/youtube-music/issues/83 - other users are getting ads too, it seems to depend on various settings (the country you are in, being logged in or not, etc), ideas for solutions are being discussed in the issue!

**Reply by matheusbc22** [reply]
I didn't saw this, sorry, i think i just solved the problem, thanks!

**Reply by LatvianBHood** [reply]
I'm having AdBlock issiues no matter what I do.

**Reply by th-ch** [reply]
Hey @LatvianBHood, thanks for the feedback! Have you tried [v1.7.4](https://github.com/th-ch/youtube-music/releases/tag/v1.7.4)? It should solve adblocking issues!

**Reply by LatvianBHood** [reply]
The update fixed it, thanks! This app is way faster than the browser version too :D

**Reply by inson1** [reply]
@matheusbc22 May you close this disscussion if its done?
